cocotb/log.py

```diff
@@ -72,17 +72,16 @@ def default_config():
     """
     # construct an appropriate handler
     hdlr = logging.StreamHandler(sys.stdout)
+    hdlr.addFilter(SimTimeContextFilter())
```

Comment:
There are some tests in test_cocotb which do something similar, should we align them for consistency, or don't we care? (I assume the latter.)

Reply:
I don't understand the question

Reply:
I now see why -- it can't make sense :-) I had modified the logging test code locally for my last logging changes and had introduced similar code. But none of that is actually in cocotb, so you can't see what I had in mind :smile: